StructureDefinition/NHSDigital-PractitionerRole-Minimal.StructureDefinition.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
   <id value="51533f7f-1fa3-4866-a200-01618d377455" />
-  <url value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-PractitionerRole-Minimal" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-PractitionerRole-Minimal" />
   <name value="NHSDigitalPractitionerRoleMinimal" />
```

Comment:
All structure definitions with NHSDigital in the name should be updated, I won't be adding comments to the rest for the same issue